static/src/inline-svgs/membership/videoCaption.svg

```diff
@@ -0,0 +1,4 @@
+
+<svg viewBox="92 875 134 69" preserveAspectRatio="none">
+    <polygon id="" stroke="none" fill="#A4D8F4" fill-rule="evenodd" points="225.106952 880.342246 225.106952 920.395722 219.764706 925.737968 120.491979 925.737968 109.718449 943.545455 106.245989 943.545455 106.245989 925.737968 97.2532086 925.737968 92 920.395722 92 880.342246 97.342246 875 219.764706 875"></polygon>
```

Comment:
Do we need `id`, `stroke`, `fill` and `file-rule` here? Can `fill` be specified in CSS?